server/templates/home.html

```diff
@@ -1,67 +1,180 @@
 <!DOCTYPE html>
 <html lang="en">
-<!-- Based off of example: https://raw.githubusercontent.com/23/resumable.js/master/samples/Node.js/public/index.html -->
+
```

Comment:
I would actually prefer to keep the comment to acknowledge the source of a lot of the code.